test/onnx/onnx_test_common.py

```diff
@@ -130,13 +201,31 @@ def _run_test(m, remained_onnx_input_idx, flatten=True, ignore_none=True):
             model, (torch.jit.ScriptModule, torch.jit.ScriptFunction)
         )
 
-        if self.is_script_test_enabled and self.is_script:
+        if self.is_script:
+            # Run test for export route of `torch.nn.Module` -> `torch.jit.ScriptModule` -> ONNX.
             script_model = model if is_model_script else torch.jit.script(model)
             _run_test(
                 script_model,
                 scripting_remained_onnx_input_idx,
                 flatten=False,
                 ignore_none=False,
             )
-        if not is_model_script and not self.is_script:
+        elif not is_model_script and not self.is_script and not self.is_fx:
```

Comment:
Can we move the elif self.is_fx on top of this and make this else? This is the tracing cases, right? I guess you think this is more explicit, but it actually seems more complicated. But not blocking though.

Reply:
Could be just me, but adding a comment would be better.

Reply:
Cleared the logic a bit.